_posts/02-01-01-Code-Style-Guide.md

```diff
@@ -1,61 +1,63 @@
 
-The PHP community is large and diverse, composed of innumerable libraries, frameworks, and components. It is common for
-PHP developers to choose several of these and combine them into a single project. It is important that PHP code adhere
-(as close as possible) to a common code style to make it easy for developers to mix and match various libraries for
-their projects.
+PHP komunita je veľká a rozmanitá. Skladá sa z nespočetného množstva knižníc, frameworkov a komponentov.
+Pre PHP programátorov je ich bežné používať a kombinovať v rámci jedného projektu niekoľko naraz. Je dôležité, aby sa
+kód PHP pridržiaval, pokiaľ možno čo najbližšie, k všeobecne zaužívanému štýlu písania kódu. Toto programátorom uľahčuje
+v ich projektoch použiť rôzne knižnice bez nutnosti použitia rôznych štýlov písania kódu v rôznich častiach.
 
-The [Framework Interop Group][fig] has proposed and approved a series of style recommendations. Not all of them related
-to code-style, but those that do are [PSR-0][psr0], [PSR-1][psr1], [PSR-2][psr2] and [PSR-4][psr4]. These
-recommendations are merely a set of rules that some projects like Drupal, Zend, Symfony, CakePHP, phpBB, AWS SDK,
-FuelPHP, Lithium, etc are starting to adopt. You can use them for your own projects, or continue to use your own
-personal style.
+Skupina [Framework Interop Group][fig] navrhla a odsúhlasila sériu doporučení písania kódu. Nie každé z týchto
+doporučení ale s týmto súvisí. Tie, ktoré sa tohoto týkajú sú [PSR-0][psr0], [PSR-1][psr1], [PSR-2][psr2]
+a [PSR-4][psr4]. Tieto doporučenia sú viacmenej zoznamom pravidiel, ktoré projekty ako napríklad Drupal, Zend,
+Symfony, CakePHP, phpBB, AWS SDK, FuelPHP, Lithium, atď. začinajú zavádzať. Tieto doporučenia môžete používať taktiež
+pre vaše projekty. Môžete ale taktiež pokračovať v použivaní vášho osobného štýlu.
 
-Ideally you should write PHP code that adheres to a known standard. This could be any combination of PSRs, or one
-of the coding standards made by PEAR or Zend. This means other developers can easily read and work with your code, and
-applications that implement the components can have consistency even when working with lots of third-party code.
+Kód PHP by ste ideálne mali písať tak, aby sa pridržiaval všeobecne známemu štandardu. Toto môže byť ľubovolná
+kombinácia doporúčení PSR, alebo jeden zo štandardov PEAR, alebo Zend. Toto zabezpečí čitateľnosť vášho kódu pre iných
+programátorov a konzistentnosť štýlu kódu v aplikáciach, ktoré môžu váš kód použivať.
 
-* [Read about PSR-0][psr0]
-* [Read about PSR-1][psr1]
-* [Read about PSR-2][psr2]
-* [Read about PSR-4][psr4]
-* [Read about PEAR Coding Standards][pear-cs]
-* [Read about Symfony Coding Standards][symfony-cs]
+* [Prečítajte si o PSR-0][psr0]
+* [Prečítajte si o PSR-1][psr1]
+* [Prečítajte si o PSR-2][psr2]
+* [Prečítajte si o PSR-4][psr4]
+* [Prečítajte si o štandarde písania kódu PEAR][pear-cs]
+* [Prečítajte si o štandarde písania kódu Symfony][symfony-cs]
```

Comment:
PEAR and ZEND coding standard is mentioned couple lines above, but missing in this list. It would be good to include it here.